.github/workflows/release.yml

```diff
@@ -0,0 +1,315 @@
+          # x86_64
+          linux-x86_64-gnu, linux-x86_64-musl, macos-x86_64, win64-x86_64-gnu, win64-x86_64-msvc
+        ]
+        include:
+        - { name: 'linux-aarch64-gnu' , target: aarch64-unknown-linux-gnu   , os: ubuntu-latest , use-cross: true }
+        - { name: 'linux-aarch64-musl', target: aarch64-unknown-linux-musl  , os: ubuntu-latest , use-cross: true }
+        - { name: 'macos-aarch64'     , target: aarch64-apple-darwin        , os: macos-latest  ,                 }
+        - { name: 'win64-aarch64-msvc', target: aarch64-pc-windows-msvc     , os: windows-latest,                 }
+        - { name: 'linux-x86_64-gnu'  , target: x86_64-unknown-linux-gnu    , os: ubuntu-latest ,                 }
+        - { name: 'linux-x86_64-musl' , target: x86_64-unknown-linux-musl   , os: ubuntu-latest , use-cross: true }
+        - { name: 'macos-x86_64'      , target: x86_64-apple-darwin         , os: macos-latest  ,                 }
+        - { name: 'win64-x86_64-gnu'  , target: x86_64-pc-windows-gnu       , os: windows-latest,                 }
+        - { name: 'win64-x86_64-msvc' , target: x86_64-pc-windows-msvc      , os: windows-latest,                 }
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Install rust
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        target: ${{ matrix.target }}
+
+    - uses: Swatinem/rust-cache@v2
+      with:
+        key: ${{ matrix.target }}
+
+    - name: Build
+      uses: actions-rs/cargo@v1
+      with:
+        use-cross: ${{ matrix.use-cross }}
+        command: build
+        args: >-
+          --package ${{ env.PROJECT_NAME }}
+          --profile ${{ env.CARGO_PROFILE }}
+          --target ${{ matrix.target }}
+          --locked --verbose
+
+    - name: Build archive
+      shell: bash
+      run: |
+        PKG_BASENAME="${{ env.PROJECT_NAME }}-v${{ env.PROJECT_VERSION }}-${{ matrix.target }}"
+        mkdir "${PKG_BASENAME}"
+
+        EXE_suffix=""
+        case ${{ matrix.target }} in
+          *-pc-windows-*) EXE_suffix=".exe" ;;
+        esac
+        PROJECT_BIN_PATH="target/${{ matrix.target }}/${{ env.CARGO_PROFILE }}/${{ env.PROJECT_NAME }}${EXE_suffix}"
+
+        cp "${PROJECT_BIN_PATH}" "${PKG_BASENAME}/"
+        cp README.md CHANGELOG.md LICENSE-MIT LICENSE-APACHE "${PKG_BASENAME}/"
+
+        case ${{ matrix.target }} in
+          *-pc-windows-*)
+              PKG_PATH="${PKG_BASENAME}.zip"
+              7z -y a "${PKG_PATH}" "${PKG_BASENAME}"/* | tail -2
+            ;;
+          *)
+              PKG_PATH="${PKG_BASENAME}.tar.gz"
+              tar czf "${PKG_PATH}" "${PKG_BASENAME}"/*
+            ;;
+        esac
+        echo "PKG_PATH=${PKG_PATH}" >> $GITHUB_ENV
+
+    - uses: actions/upload-artifact@v3
+      with:
+        name: ${{ env.PKG_PATH }}
+        path: ${{ env.PKG_PATH }}
+        retention-days: 1
+
+  github:
+    runs-on: ubuntu-latest
+    needs: [check, binary]
```

Comment:
Worth highlighting that the release on GitHub doesn't depend on the crate release.

Yet it includes the crate link in the release notes.

What do you think? Should we wait for the crate to be published, or not?